lib/local-storage-adapter.js

```diff
@@ -130,20 +129,31 @@ function createLocalStorageAdapter (queueName) {
     adapter.remove(queueKey)
     adapter.remove(queueProcessingKey)
   }
+}
 
-  function save (key, data) {
-    window.localStorage[key] = data
-  }
+function save (key, data) {
+  window.localStorage[key] = data
+}
 
-  function load (key) {
-    return window.localStorage[key]
-  }
+function load (key) {
+  return window.localStorage[key]
+}
 
-  function remove (key) {
-    window.localStorage.removeItem(key)
-  }
+function remove (key) {
+  window.localStorage.removeItem(key)
+}
+
+function now () {
+  return (new Date()).getTime()
+}
 
-  function now () {
-    return (new Date()).getTime()
+function onceOnDefer (fn) {
+  var called = false
+  return function debounced () {
+    if (called) return
+    called = true
+    setTimeout(function later () {
+      fn()
```

Comment:
Are you suggesting this entire function could be replaced? I guess with setPending you could, the only difference is it stops us from deferring loads of flush calls that do nothing. Would be neater though, let me play with it.